### PR TITLE
Add details of the other models to the readme

### PR DESCRIPTION
**Pull request by sbonner0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,9 @@ The code relies primarily on the [PyKEEN](https://github.com/pykeen/pykeen) pack
 
 A pretrained version of the [TransE](https://proceedings.neurips.cc/paper/2013/file/1cecc7a77928ca8133fa24680a88d2f9-Paper.pdf) model is provided in the `artifacts` directory. This version is automatically used in the notebooks.
 
-However, code is provided to retrain the model or swap it out for a different one in the `train.py` file in the `src` directory. Please use this function to retrain the other models evaluated in the study.
+Due to the size of weight files for the other models used in the study, they were unable to be included in main repository. However, the files are available for download as part of the release of the repository [here](https://github.com/AstraZeneca/biomedical-kg-topological-imbalance/releases/tag/0.2.0). Once the files have been downloaded, please place them in the `artifacts` directory if you wish to use them.
```
Comment (cthoyt) [inline README.md:38]: would be better to 
1. archive on zenodo
2. Include automatic download of these resources (e.g., using `pystow`) in the code itself

Comment (sbonner0) [reply README.md:38]: Thanks @cthoyt this is certainly a better solution! I'll look into getting weights onto zenodo and then have`pystow` manage the download as need. Can I reach out for tips regarding best practices for `pystow` and any help if needed?

Comment (cthoyt) [reply README.md:38]: yes would be happy to help :)

Comment (cthoyt) [reply README.md:38]: I also wrote https://github.com/cthoyt/zenodo-client/ on top of pystow for making it easier to get data specifically off of zenodo using pystow

**Pull request merged**